src/Microsoft.VisualStudio.LanguageServices.Razor/DefaultTagHelperResolver.cs

```diff
@@ -2,27 +2,37 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.AspNetCore.Mvc.Razor.Extensions;
-using Microsoft.AspNetCore.Razor.Language;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Razor;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
+using InProcessTagHelperResolver = Microsoft.VisualStudio.Editor.Razor.DefaultTagHelperResolver;
 
 namespace Microsoft.VisualStudio.LanguageServices.Razor
 {
     internal class DefaultTagHelperResolver : TagHelperResolver
     {
+        private readonly InProcessTagHelperResolver _inProcessResolver;
         private readonly ErrorReporter _errorReporter;
         private readonly Workspace _workspace;
 
         public DefaultTagHelperResolver(ErrorReporter errorReporter, Workspace workspace)
         {
+            if (errorReporter == null)
+            {
+                throw new ArgumentNullException(nameof(errorReporter));
+            }
+
+            if (workspace == null)
+            {
+                throw new ArgumentNullException(nameof(workspace));
+            }
+
             _errorReporter = errorReporter;
             _workspace = workspace;
+            _inProcessResolver = new InProcessTagHelperResolver(_errorReporter);
```

Comment:
A little icky doing this but didn't want to use inheritance either. Feel free to suggest something else.

Reply:
I think this is fine.